docs/img/logomark_purser_green.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This SVG was
- not flatten
- has decimal point viewbox sizing
- has a bunch of useless metadata

Please use this one:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 86 114" version="1.1">
  <path style="fill:#19a582" d="M 14.583984 0 L 43.050781 28.273438 L 71.416016 0 L 14.583984 0 z M 42.949219 32.986328 C 19.211206 32.986328 -4.7369516e-15 52.137961 0 72.892578 L 0 114 L 86 114 L 86 75.900391 C 86 52.137859 66.787816 32.986328 42.949219 32.986328 z M 43.050781 57.150391 C 52.002829 57.150391 59.345703 64.469096 59.345703 73.392578 C 59.345703 82.416325 52.001851 89.736328 42.949219 89.736328 C 33.896587 89.736328 26.654297 82.416325 26.654297 73.392578 C 26.654297 64.368832 33.998149 57.150391 43.050781 57.150391 z" />
</svg>
```